### PR TITLE
AOM-37: Additional fix; Make imports consistent, improve code standard.

### PR DESCRIPTION
**Pull request by kingisaac95**
## JIRA TICKET NAME:
AOM-37: [Remove dashboard and link user directly to manage add on page](https://issues.openmrs.org/browse/AOM-37)
### SUMMARY:
Make imports consistent, improve code standard.

### CODE REVIEW

**Comment by kingisaac95** [comment]
@andela-jomadoye @paulupendo @wanjikum Please review this so it can be merged.

**Comment by jomadoye** [comment]
Good job @kingisaac95

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -11,7 +11,6 @@ import axios from 'axios';
 import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
 import AddAddon from '../manageApps/AddAddon.jsx';
 import BreadCrumbComponent from '../breadCrumb/BreadCrumbComponent.jsx';
-import Header from '../common/Header';
 import { ApiHelper } from '../../helpers/apiHelper';
 import { AddonList } from './AddonList.jsx';
 import DeleteAddonModal from './DeleteAddonModal.jsx';
```
Comment (wanjikum) [inline app/js/components/manageApps/ManageApps.jsx]: Could you remove all the .jsx extensions in this file?

**Comment by wanjikum** [comment]
@kingisaac95 Could you upload the changes into the modules ref app?

**Comment by wanjikum** [comment]
nice one 👍

**Comment by wanjikum** [comment]
Why did you remove the border-bottom in the breadcrumb.css file? I believe it should be there...

**Comment by kingisaac95** [comment]
@wanjikum it's disrupting the UI flow - unnecessary I'd say, @andela-jomadoye saw how it was.

**Comment by kingisaac95** [comment]
@dkayiwa Please can you merge this?

**Pull request merged**